app/charts/bar/bars-state.tsx

```diff
@@ -64,9 +65,10 @@ const useBarsState = ({
   const getY = useStringVariable(fields.y.componentIri);
   const getSegment = useSegment(fields.segment?.componentIri);
 
-  const xAxisLabel =
-    measures.find((d) => d.iri === fields.x.componentIri)?.label ??
-    fields.y.componentIri;
+  const xMeasure = measures.find((d) => d.iri === fields.x.componentIri);
+
+  const xAxisLabel = xMeasure?.label ?? fields.y.componentIri;
+  const xAxisDescription = xMeasure?.description as string | undefined;
```

Comment:
Nit: Not sure the type assertion is good since we can have null right ?

```
xMeasure?.description || undefined
```

achieves the same thing without type assertion.